scripts/install-multi-user.sh

```diff
@@ -576,18 +576,37 @@ create_directories() {
         # since this bit is cross-platform:
         # - first try with `command -vp` to try and find
         #   chown in the usual places
+        #   * to work around some sort of deficiency in
+        #     `command -p` in macOS bash 3.2, we also add
+        #     PATH="$(getconf PATH 2>/dev/null)". As long as
+        #     getconf is found, this should set a sane PATH
+        #     which `command -p` in bash 3.2 appears to use.
+        #     A bash with a properly-working `command -p`
+        #     should ignore this hard-set PATH in favor of
+        #     whatever it obtains internally. See
+        #     github.com/NixOS/nix/issues/5768#issuecomment-1018060400
```

Comment:
Nit: Don't you think that the line length looks really silly here?

Reply:
Not sure if you meant just the link, or broadly; I was mostly just breaking to avoid splitting commands over a linebreak, and then maintained it. If the former, I dropped the anchor from the link, which I suppose _looks_ a little less silly.